components/filters/FilterPicker.tsx

```diff
@@ -58,7 +58,7 @@ export default function FilterPicker({
     <>
       <div className="picker">
         <div className="picker__selection">
-          <DataList gap="0rem 3rem">
+          <DataList gap="1rem 3rem">
```

Comment:
I'm trying to make sense of what this changes 😅 I'm flipping between the two versions and it isn't super apparent that anything is changing. Happy to just let this one fly but I'm mostly just curious.

Reply:
Intention was to space out the filters on smaller screen sizes so there was less error for unintentionally tapping on another filter

**Before**:
<img width="352" alt="Screen Shot 2022-12-13 at 7 46 20 AM" src="https://user-images.githubusercontent.com/26368729/207379963-9d0769a0-d147-41fd-af2f-4e512717f707.png">

**After**:
<img width="353" alt="Screen Shot 2022-12-13 at 7 46 30 AM" src="https://user-images.githubusercontent.com/26368729/207380028-e31775cb-b3eb-4368-9e7c-b6ce3e12378b.png">

Reply:
Thinking now though, I could also add functionality in DataList to only have this vertical spacing be applied for smaller screens, let me know